substrate/frame/parameters/src/lib.rs

```diff
@@ -184,7 +202,12 @@ pub mod pallet {
 	}
 
 	/// Stored parameters.
+	///
+	/// ## Storage Whitelist
+	/// Since we account for all parameters read weight for each block,
+	/// don't double count it in other pallet benchmarks
 	#[pallet::storage]
+	// #[pallet::whitelist_storage]
```

Comment:
Do not forget to update this part
```suggestion
	#[pallet::storage]
```